include/picongpu/plugins/particleCalorimeter/ParticleCalorimeter.hpp

```diff
@@ -181,12 +194,30 @@ namespace picongpu
 
             hdf5DataFile.open(filename.str().c_str(), fAttr);
 
+            filename = std::stringstream{};
+            filename << checkpointDirectory << "/" << this->foldername << "/" << filenamePrefix << "_%T."
+                     << filenameExtension;
+            ::openPMD::Series series(filename.str(), ::openPMD::Access::CREATE);
+
             typename PICToSplash<float_X>::type SplashTypeX;
```

Comment:
just a question: should/will this be replaces in the future or in other words is PICToSplash maintained in the future?

Reply:
This line will go once I remove the libsplash output in this PR, the openPMD replacement is `::openPMD::determineDatatype<float_X>()`.

Reply:
This will be removed too, I would say we have a coresponding helper for OpenPMD https://github.com/ComputationalRadiationPhysics/picongpu/blob/454a28efe6a6eb0fa1e0e97031ef2569a1ad313b/include/picongpu/traits/PICToOpenPMD.hpp